pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -354,7 +354,10 @@ func (lb *ListenerBuilder) buildHTTPConnectionManager(httpOpts *httpListenerOpts
 	routerFilterCtx, reqIDExtensionCtx := configureTracing(lb.push, lb.node, connectionManager, httpOpts.class)
 
 	filters := []*hcm.HttpFilter{}
-	wasm := lb.push.WasmPlugins(lb.node)
+	wasm := lb.push.WasmPluginsByListenerInfo(lb.node, &model.WasmPluginListenerInfo{
```

Comment:
```suggestion
	wasm := lb.push.WasmPluginsByListenerInfo(lb.node, model.WasmPluginListenerInfo{
```

No need for pointer, just introduces more possibility for errors